tellor_all_languages/test/test_file_contents_changed.js

```diff
@@ -0,0 +1,194 @@
+var urllib = require('urllib');
+
+const CompareFileContents = artifacts.require("./CompareFileContents.sol");
+const Tellor = artifacts.require("TellorPlayground.sol");
+
+//Helper function that submits and value and returns a timestamp for easy retrieval
+const submitTellorValue = async (tellorOracle, requestId, amount) => {
+  //Get the amount of values for that timestamp
+  let count = await tellorOracle.getNewValueCountbyRequestId();
+  await tellorOracle.submitValue(requestId, amount);
+  let time = await getTimestampbyRequestIDandIndex(requestId, count.toString());
+  return time.toNumber();
+};
+
+contract("UsingTellor Tests", function (accounts) {
+	let compareFileContents;
+	let tellorOracle;
+
+	beforeEach("Setup contract for each test", async function () {
+	tellorOracle = await Tellor.new();
+	compareFileContents = await CompareFileContents.new(tellorOracle.address);
+	});
+
+	it("Update Price", async function () {
+	const requestId = 1; // assumes the first ID of the list of tellor variable contains what we return, e.g. usd/BTC (in our case, checkflag text)
+	// TODO: change the requestId to two strings: 
+	// 0. the bounty hunter github name
+	// 1. The repository name and commit belonging to the sponsor contract 
+	// OR let the Tellor oracles scrape this data from the sponsor contract to reduce gas costs.
+	// (unless the Tellor oracles cannot find what the bounty hunter contract was that initated this Tellor query,
+	// in that case the repository name and commit of the bounty hunter should be passed).
+    	
+    	
+	// -----------------------------------------Helper Functions ----------------------------
+	// Encode a string to a number 
+	// Source: https://stackoverflow.com/questions/14346829/is-there-a-way-to-convert-a-string-to-a-base-10-number-for-encryption
+	function encode(string) {
+		var number = "0x";
+		var length = string.length;
+		for (var i = 0; i < length; i++)
+			number += string.charCodeAt(i).toString(16);
+		return number;
+	}    	
+    	
+    // Function that runs some incoming shell command (not bash)
+	const exec = require('child_process').exec;
+	function os_func() {
```

Comment:
We should move this to a `utils` file if we're going to be using it in separate files